MetaCocktailsSwiftData/Model/SearchTags/Authors.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  Authors.swift
+//  MetaCocktailsSwiftData
+//
+//  Created by James Menkal on 1/20/24.
+//
+
+import Foundation
+
+enum Authors: String, Codable, CaseIterable {
+    
+    case deathAndCo        = "Death & Co"
+    case williamsAndGraham = "Williams and Graham"
+    case jamesMenkal       = "James Menkal"
+    case samRoss           = "Sam Ross"
+    case milkAndHoney      = "Milk and Honey"
+    case philWard          = "Phil Ward"
+    case jerryThomas       = "Jerry Thomas"
+    
+    
```

Comment:
The author part of the model is a bit messy right now. Sometimes it's an author, sometimes its a bar, sometimes its a place. I think a better solution is to make Author a struct with a name, cocktailBar, and year. All optional. We don't need to use a place like New Orleans, if the author is unknown that's ok. 

Then we can make a presentationString variable on the struct as well that looks at what it has and returns the string we'll use on the cocktail page.
So depending on the author struct, calling Author.jamesMenkal.presentationString will return "James Menkal" or "James Menkal, Williams and Graham" or "James Menkal, Williams and Graham (2019)" depending on what information is available in the struct